src/components/useraccount/orderdetail/PriceCalculation.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <div v-if="me"
+       class="my-orders-order-price-summary-wrapper">
+    <!-- <div v-if="me.order.discountCodes.length > 0" class="text-right">
+      <span class="my-orders-order-promocode-title">
+        {{ $t('discountCode') }}:
+      </span>
+      <span class="my-orders-order-promocode">
+        "{{me.order.discountCodes[0].discountCode.code}}"
+      </span>
+    </div> -->
+    <div class="text-right">
+      <div class="row">
+        <div class="col-sm-10 col-xs-7">
+          <div class="text-right subtotal">
+            <span class="subtotal-title">{{ $t('subtotal') }}</span>
+          </div>
+          <!-- <div v-if="me.order.discountCodes.discountCode.isActive"
+            class="text-right">
+            <span class="order-discount">{{ $t('orderDiscount') }}</span>
+          </div>-->
+          <div class="text-right delivery-info">
+            <span class="delivery-info-title">{{ $t('shipping')}}</span>
+          </div>
+          <hr class="total-divider">
+          <div class="text-right">
+            <span>{{ $t('salesTax') }}</span>
+          </div>
+          <div class="text-right">
+            <span class="order-total">{{ $t('total') }}</span>
+          </div>
+        </div>
+        <div class="col-sm-2 col-xs-5 text-right">
+          <div data-test="order-subtotal">
+            <BaseMoney :money="subtotal"/>
+          </div>
+          <!-- <div v-if="me.order.discountCodes.discountCode.isActive">
+          <span class="order-discount">{{cart.discount}}</span>
+          </div>-->
+          <div>
+            <BaseMoney :money="me.order.shippingInfo.price"/>
+          </div>
+          <hr>
+          <div>
+            <BaseMoney :money="me.order.taxedPrice.taxPortions[0].amount"/>
```

Comment:
It may be that in the old Sunrise it was showing only one tax portions, but we should show all of them. In general, having `[0]` anywhere it may be an indication we are missing cases and it may also fail if no element is found, so make sure you only call that into very justified situations :)